docs/docs/external/getting-started/test-network/setup-local.md

```diff
@@ -134,6 +134,10 @@ Follow the instructions below to build and launch the networks:
   ```bash
   make start-interop-local CHAINCODE_NAME=<chaincode-name>
   ```
+- _To launch the networks with 2 orgs, each with a peer (this will enable more variation and experimentation, which you can attempt after testing interoperation protocols across basic network configurations), run_:
```

Comment:
Again, maybe change "orgs" to "organizations" here as this is the first mention in this page (please ignore if not required)?

Reply:
Done